### PR TITLE
REL-931516 Server 2023 Patch 2 - Update RAPCI repos to upgrade hopper with hotfix/patch.

### PR DESCRIPTION
**Pull request by shivarajkamadod**
- Propagating patch1 changes to server release 2023
- [REL-931516](https://jira.kcura.com/browse/REL-931516) Server 2023 Patch 2 - Update RAPCI repos to upgrade hopper with hotfix/patch.
- 
image

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,12 +2,31 @@
 
 All notable changes to this project will be documented in this file.
 
+## [23012.8.1003] - 01-APR-2024
```
Comment (ravikonnur) [inline CHANGELOG.md:5]: Modify the date to May 2024

File: README.md [hunk 2]
```diff
@@ -17,9 +17,9 @@ For functional tests, point PowerShell to the root of this repository and provid
 
 ## History  
 
-The server-release-12.3-2023 branch was migrated from https://git.kcura.com/projects/DTX/repos/kcura.singlefileupload/browse?at=refs%2Fheads%2Fserver-release-12.3-2023 by the Server Data Transfer Team.
-Tag: 23012.8.3
-Branch: server-release-12.3-2023
+The server-release-12.3-2023-patch1 branch was migrated from https://git.kcura.com/projects/DTX/repos/kcura.singlefileupload/browse?at=refs%2Fheads%2Fserver-release-12.3-2023-patch1 by the Server Data Transfer Team.
```
Comment (ravikonnur) [inline README.md:20]: This information should not be changed

Comment (shivarajkamadod) [reply README.md]: Done

File: CHANGELOG.md (hunk 1 above)
Comment (shivarajkamadod) [reply CHANGELOG.md]: Done

**Review by ravira73 — APPROVED** [review]

**Pull request merged**